test.py

```diff
@@ -118,3 +120,39 @@ def test_nested_input():
 def test_function_mixup():
     """Make sure that different functions with the same args are cached under different keys"""
     assert get_random_with_args(1) != get_another_random_with_args(1)
+
+
+def callTextChangeTest():
```

Comment:
1. PEP8 - `call_text_change_test` - https://www.python.org/dev/peps/pep-0008/#function-and-variable-names
2. I should understand what the function does from its name. This one is a bit unclear.